src/lib/udev/rules.d/65-gce-disk-naming.rules

```diff
@@ -21,7 +21,30 @@ SUBSYSTEM!="block", GOTO="gce_disk_naming_end"
 KERNEL=="sd*|vd*", IMPORT{program}="scsi_id --export --whitelisted -d $tempnode"
 
 # NVME Local SSD naming
-KERNEL=="nvme*n*", ATTRS{model}=="nvme_card", PROGRAM="/bin/sh -c 'echo $$((%n-1))'", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-%c"
+KERNEL=="nvme0n1*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-0"
+KERNEL=="nvme0n2*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-1"
+KERNEL=="nvme0n3*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-2"
+KERNEL=="nvme0n4*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-3"
+KERNEL=="nvme0n5*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-4"
+KERNEL=="nvme0n6*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-5"
+KERNEL=="nvme0n7*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-6"
+KERNEL=="nvme0n8*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-7"
+KERNEL=="nvme0n9*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-8"
+KERNEL=="nvme0n10*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-9"
+KERNEL=="nvme0n11*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-10"
+KERNEL=="nvme0n12*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-11"
+KERNEL=="nvme0n13*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-12"
+KERNEL=="nvme0n14*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-13"
+KERNEL=="nvme0n15*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-14"
+KERNEL=="nvme0n16*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-15"
+KERNEL=="nvme0n17*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-16"
+KERNEL=="nvme0n18*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-17"
+KERNEL=="nvme0n19*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-18"
+KERNEL=="nvme0n20*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-19"
+KERNEL=="nvme0n21*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-20"
+KERNEL=="nvme0n22*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-21"
+KERNEL=="nvme0n23*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-22"
+KERNEL=="nvme0n24*", ATTRS{model}=="nvme_card", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-23"
```

Comment:
i will review this with you more in depth tomorrow, but the previous change was designed to remove explicit rules like these

Reply:
I tried the following as a one line solution, but it didn't work. Seems like udev doesn't have a good support on escaping double quotes https://github.com/systemd/systemd/issues/6835. This PR is made as a quick and easy fix.

```
KERNEL=="nvme*n*", ATTRS{model}=="nvme_card", PROGRAM="echo %k | sed \"s/nvme[0-9]\+n\([0-9]\+\).*/\1/g\"", ENV{ID_SERIAL_SHORT}="local-nvme-ssd-%c"
```